src/components/wallet/methods/claimAsset.ts

```diff
@@ -20,6 +20,11 @@ export default async function claimAsset(
   this.loading = { ...this.loading, [loadingKey]: true }
   const finish = () => (this.loading = { ...this.loading, [loadingKey]: false })
   try {
+    if (!this.balance.get(balance.asset).trusted) {
+      let asset = this.balance.get(balance.asset).asset_code
+      let issuer = this.balance.get(balance.asset).asset_issuer
```

Comment:
Just to make sure I understand correctly, these two lines could be replaced with:
```
let [asset, issuer] = balance.asset.split(":")
```
right?

Reply:
Oh great point!